README.md

```diff
@@ -51,13 +51,17 @@ Install the packages from `requirements.txt`
 $ pip install -r requirements.txt
 ```
 
+## About the environment
+
+This package simulates an oscillating bridge by modelling it with the one-dimensional [wave equation](https://en.wikipedia.org/wiki/Wave_equation), which is simulated using a [finite difference solver](https://en.wikipedia.org/wiki/Finite_difference_method).  The action space of the environment represents pistons that apply a force to actively dampen vibrations in the bridge.  The reward signal is inversely proportional to the interal energy of the system.  One episode of the environment involves 3 phases:  1) A "warmup phase" where an external force is applied to the system to cause oscillations 2) An "equilibriation" phase where the oscillations settle in to stable patterns and 3) A dampening phase where the agent attempts to dampen the oscillations.
+
 ## Training an agent
 
-To train an agent for 1000 timesteps on the vibrating bridge environment and save it as 'my_first_agent.pkl', navigate to the `complex_envs` folder and run the following command:
+To train an agent for 1000 timesteps on the vibrating bridge environment and save it as `my_first_agent.pkl`, navigate to the `complex_envs` folder and run the following command:
 
 ```
 $ cd complex_envs
```

Comment:
python code should not be in the envs folder. That should just be the envs. Code should all be run from the root dir, and train etc. code should be in a different dir, e.g. python

Reply:
Understood.  I am aiming to do a large refactor/cleanup of the directory this weekend, since we aren't demoing next week.